src/wp-includes/post.php

```diff
@@ -8018,3 +8019,29 @@ function use_block_editor_for_post_type( $post_type ) {
 	 */
 	return apply_filters( 'use_block_editor_for_post_type', true, $post_type );
 }
+
+/**
+ * Registers any additional post meta fields.
+ *
+ * @since 6.3.0 Adds sync_status meta field to the wp_block post type so an unsynced option can be added.
+ *
+ * @link https://github.com/WordPress/gutenberg/pull/51144
+ */
+function wp_create_initial_post_meta() {
+	$post_type = 'wp_block';
+	register_post_meta(
+		$post_type,
```

Comment:
```suggestion
	register_post_meta(
		'wp_block',
```

This is mainly to be defensive now the function is been set up as a location to register all the things later.

Reply:
Makes sense, thanks